src/www/status_graph.php

```diff
@@ -348,6 +362,9 @@ function update_bandwidth_stats() {
                         <option value="local" <?=$pconfig['filter'] == "local" ? " selected=\"selected\"" : "";?>>
                           <?= gettext('Local') ?>
                         </option>
+                        <option value="private" <?=$pconfig['filter'] == "private" ? " selected=\"selected\"" : "";?>>
```

Comment:
The  `selected=\"selected\""` could go  (on the existing options too), they can't be all selected at the same time so it will pick the first one anyway.